gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/DiskStoreImpl.java

```diff
@@ -4811,16 +4814,16 @@ public void run() {
           SystemFailure.checkFailure();
           indexRecoveryFailure.compareAndSet(null, err);
         } finally {
-          for (Oplog oplog : this.allOplogs) {
-            oplog.clearInitRecoveryMap();
-          }
+//          for (Oplog oplog : this.allOplogs) {
+//            oplog.clearInitRecoveryMap();
+//          }
```

Comment:
This change might not be required at all because recreate is based on reading the crf and the recoverymap is not consulted at all. Will check and remove it.